src/ipv4.h

```diff
@@ -76,7 +77,7 @@ struct ipv4_config {
 	struct rtentry	def_rt; // default route
 	struct rtentry	gtw_rt; // route to access VPN gateway
 	struct rtentry	ppp_rt; // new default route through VPN
-	struct rtentry	split_rt[MAX_SPLIT_ROUTES]; // split VPN routes
+	struct rtentry	*split_rt; // split VPN routes
```

Comment:
Do we have to free this explicitly at the end?

Reply:
It's probably better to free explicitly - might be handy in case we add the feature where _openfortivpn_ reconnects automatically after a time-out.